redi-connect-front/src/components/molecules/FilterDropdown.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect, useRef } from 'react'
+import { Checkbox, Icon } from '../atoms'
+import classnames from 'classnames'
+import './FilterDropdown.scss'
+
+interface Props {
+  label: string
+  selected: string[]
+  items: {label: string, value: string}[]
+  onChange: (item: string) => void
+}
+
+const FilterDropdown = ({ items, selected, label, onChange }: Props) => {
```

Comment:
`HTMLDivElement` instead of `any`

Reply:
I was struggling soooo long before I wrote `any` with this, but found no solution. `HTMLDivElement` does not work on the ref for some weird reason.

Reply:
Weird. Wild guess, try to pull my changes, nuke `node_modules`, re-run `yarn`?